scripts/smoke-test.sh

```diff
@@ -1,19 +1,38 @@
 #!/bin/bash
 
-NAMESPACE="${NAMESPACE:-kubefed-test}"
+#Default values
+NAMESPACE="${NAMESPACE:-default}"
 LOCATION="${LOCATION:-local}"
 VERSION="${VERSION:-v0.1.0-rc2}"
+IMAGE_NAME="${IMAGE_NAME:-quay.io/sohankunkerkar/kubefed-operator:v0.1.0}"
+SCOPE="${SCOPE:-Namespaced}"
+while getopts “n:d:i:s:” opt; do
+    case $opt in
+	n) NAMESPACE=$OPTARG ;;
+	d) LOCATION=$OPTARG ;;
+  i) IMAGE_NAME=$OPTARG ;;
```

Comment:
formatting of i and s optargs?

Reply:
strange!. In vscode, I don't see any issues with the formatting. Let me change it.